### PR TITLE
Update contactus.html

### PR DESCRIPTION
**Pull request by Sumindar**
some updations.

### CODE REVIEW

File: contactus.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h1 id ="contact">CONTACT US THROUGH MAIL</h1>
           <label for="inputEmail3" class="col-sm-2 col-form-label">Email:</label>
           <div class="col-sm-10">
             <input type="email" class="form-control" id="inputEmail3" placeholder="abc@gmail.com">
-            <small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.</small>
+            <small id="emailHelp" class="form-text text-muted">We'll never share your email with anyone else.This is a promise.</small>
```
Comment (suhailsinghbains) [inline contactus.html]: Please add a white-space between '.' & 'T'

File: contactus.html [hunk 2]
```diff
@@ -97,7 +97,7 @@ <h1 id ="contact">CONTACT US THROUGH MAIL</h1>
                  fieldsMissing += "Please enter your password";
                }
               if(isEmail($("#inputEmail3").val()) == false){
-                errorMessage += "<p>Your email address is not valid</p>"
+                errorMessage += "<p>Your email address is not valid.PLease try again.</p>"
```
Comment (suhailsinghbains) [inline contactus.html]: Please add a white-space between '.' & 'P'
Replace 'L'  to 'l' in 'PLease'

**Review by suhailsinghbains — CHANGES_REQUESTED** [review]

**Comment by suhailsinghbains** [comment]
Thanks for the PR, I have suggested some changes, please make those, feel free to ask here for any queries.

**Comment by Sumindar** [comment]
Ok thanks suhailsinghbains.

**Comment by suhailsinghbains** [comment]
Hi,
LGTM, thanks for the effort made!
Please Edit this [comment](https://github.com/Moz-NIEC/Site-on/pull/3#issuecomment-427078430) and remove "Sir". Please don't call "Sir" or "Ma'am" in the open-source community. Always refer to people in the open-source community by either using IRC Nick or First Name.
Great work though :)

**Comment by Sumindar** [comment]
Hi,
I am a beginner in open source,so kindly guide me as to how contribute more
and more to the open source.
Thanks a lot.

**Review by suhailsinghbains — APPROVED** [review]

**Comment by suhailsinghbains** [comment]
I feel @championshuttler can guide you better. This is his [Telegram](https://t.me/championshuttler )

**Pull request merged**